src/examples/table/TableScroll.tsx

```diff
@@ -7,14 +7,18 @@
 
 import React from 'react';
 import styled from 'styled-components';
-import getScrollbarSize from 'dom-helpers/scrollbarSize';
+import { getColor } from '@zendeskgarden/react-theming';
 import { Body, Cell, Head, HeaderCell, HeaderRow, Row, Table } from '@zendeskgarden/react-tables';
 
-const SCROLLBAR_SIZE = getScrollbarSize();
+const StyledStickyHead = styled(Head)`
+  position: sticky;
+  top: 0;
+  box-shadow: ${props => {
+    const boxShadow = props.theme.shadows.sm(getColor('neutralHue', 200, props.theme) as string);
 
-const StyledSpacerCell = styled(HeaderCell)`
-  padding: 0;
-  width: ${SCROLLBAR_SIZE}px;
+    return `${boxShadow.replace('2px', '1px')}`;
+  }};
+  background-color: ${props => props.theme.colors.background};
```

Comment:
The `box-shadow` here should be expressed as a themed [border](https://6388e41c6b71d01c2d65dcb5--zendeskgarden.netlify.app/components/theme-object#borders), since that is what it is acting as. Also, there is a problem with the thickened border under the header.

Reply:
I removed the border under the `Header` to correct for the thickened border. I also updated the `box-shadow` to follow the existing example from the `Table` pattern.